### ISSUE TITLE
Add demo papers to demo_papers.csv

### ISSUE THREAD

**Issue by hao-fang** [acl-org/acl-2020-virtual-conference#72]
**Number of Volunteers**: 1

Follow-up on #8, we will need to add demo papers to the `papers.csv` file.

- [x] Get information from demo chairs (after #74 is done)

**Comment by georgepar** [comment]
@hao-fang Hi, I can do this. Is there an google sheet about demo papers available yet?

**Comment by georgepar** [comment]
@hao-fang are demo papers considered main conference papers? So do we just add them to papers.csv or we do a separate page?

**Comment by hao-fang** [comment]
Closing this with #171
I opened a follow-up issue #172.
